streams/src/commonMain/kotlin/com/mirego/trikot/streams/reactive/JustPublisher.kt

```diff
@@ -0,0 +1,11 @@
+package com.mirego.trikot.streams.reactive
+
+import org.reactivestreams.Publisher
+import org.reactivestreams.Subscriber
+
+class JustPublisher<T>(private val value: T) : Publisher<T> {
+    override fun subscribe(s: Subscriber<in T>) {
+        s.onNext(value)
+        s.onComplete()
```

Comment:
Should you check for "cancellation" on the subscription before calling "onComplete()"